src/main/java/com/wavefront/SpanQueue.java

```diff
@@ -14,16 +14,30 @@
 @ThreadSafe
 public class SpanQueue {
```

Comment:
I'll suggest renaming SpanQueue to something general, as some cases it works as TraceQueue